ion-schema/src/test/kotlin/com/amazon/ionschema/ResourceAuthorityTest.kt

```diff
@@ -68,6 +68,9 @@ class ResourceAuthorityTest {
             .build()
         val islSchema = iss.loadSchema("isl/schema.isl")
 
-        assertNotNull("Unable to find the schema for 'schema'", islSchema.getType("schema"))
+        assertNotNull(
+            islSchema.getType("schema"),
+            "Unable to find the schema for 'schema'",
+        )
```

Comment:
Parameterize `"schema"` here and you'd have a useful custom assertion. Is this the only place that we assert we're able to find a type in a schema?